### DISCUSSION TITLE
update options

### DISCUSSION THREAD

**Discussion by SebastianNikken30** [sa-si-dev/virtual-select#164] [Q&A] [answered]
Hello, how can I update the options of a previously made record? That is, to show the ones that were already selected and to be able to add or remove options.

**Comment by sa-si-dev** [comment, marked as answer]
> Hello, how can I update the options of a previously made record? That is, to show the ones that were already selected and to be able to add or remove options.

Pass second argument as `true` while calling `setOptions()` method as mentioned in the [document](https://sa-si-dev.github.io/virtual-select/#/methods?id=setoptions)
